### ISSUE TITLE
"You are not allowed to perform this operation." even with admin level permissions

### ISSUE THREAD

**Issue by KarlB25** [UiPath/uipathcli#110]
My app has the OR.Administration scope though I have tried this with individual scopes too with no joy.

uipath orchestrator users get works fine. I've also tested uipath orchestrator roles get which works.

However when trying other commands I get an authorization error. The app has been added to the Orchestrator application with admin level permissions in addition to the admin scope. Details below:

```
PS C:\WINDOWS\system32> uipath orchestrator processes get
{
  "__abp": true,
  "error": {
    "code": 0,
    "details": "You are not allowed to perform this operation.",
    "message": "You are not authorized!",
    "validationErrors": null
  },
  "errorCode": 0,
  "message": "You are not authorized!",
  "result": null,
  "success": false,
  "targetUrl": null,
  "unAuthorizedRequest": true
}
PS C:\WINDOWS\system32> uipath orchestrator folders get
{
  "__abp": true,
  "error": {
    "code": 0,
    "details": "You are not allowed to perform this operation.",
    "message": "You are not authorized!",
    "validationErrors": null
  },
  "errorCode": 0,
  "message": "You are not authorized!",
  "result": null,
  "success": false,
  "targetUrl": null,
  "unAuthorizedRequest": true
}
```

**Comment by KarlB25** [comment]
Any idea what might be causing this?

**Comment by thschmitt** [comment]
`uipath orchestrator processes get` should work with the `OR.Execution.Read` scope. If you use a confidential external application, you have to make sure that you add the `OR.Execution.Read` to the Application Scope(s):

![image](https://github.com/UiPath/uipathcli/assets/92723222/0bb065c8-da9d-427a-8f53-1195dcc649b7)

**Comment by KarlB25** [comment]
Thanks for the quick response. I tried adjusting the permissions like you said but it still failed. After that I tried creating a fresh app but assigning the OR.Execution.Read and then it worked. After that I tried "users get" but it failed. Tried adding the scope and it failed again. Next I again created a fresh app but added both scopes at the beginning and then both "processes get" and "users get" worked perfectly. Seems like assigning the app the scopes before creation works fine but adjusting scopes afterwards can create issues so just wanted to provide this feedback but happy it's now working!
